modules/nf-core/bowtie/align/main.nf

```diff
@@ -2,10 +2,7 @@ process BOWTIE_ALIGN {
     tag "$meta.id"
     label 'process_high'
 
-    conda "bioconda::bowtie=1.3.0 bioconda::samtools=1.16.1"
-    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/mulled-v2-ffbf83a6b0ab6ec567a336cf349b80637135bca3:c84c7c55c45af231883d9ff4fe706ac44c479c36-0' :
-        'biocontainers/mulled-v2-ffbf83a6b0ab6ec567a336cf349b80637135bca3:c84c7c55c45af231883d9ff4fe706ac44c479c36-0' }"
+    conda "modules/nf-core/bowtie/align/environment.yml"
```

Comment:
Would not using `bioconda::bowtie=1.3.0 bioconda::samtools=1.16.1` make it more readable?

Reply:
Main thing is wanting to incorporate Renovate to auto update or packages or us. It supports conda envs out of the box, it doesn't support a line of conda packages.